interop/src/clients/InteropClient/create_simulator.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+
+SIMULATOR_NAME="Interop Simulator - `uuidgen`"
+SIMULATOR_UDID=`xcrun simctl create "$SIMULATOR_NAME" "iPhone 16"`
```

Comment:
Is `UDID` a typo, shouldn't it be `UUID`?

Reply:
I also thought so at first but it stand for **U**nique **D**evice **ID**entifier